filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobFileSystemManager.scala

```diff
@@ -96,12 +95,16 @@ case class WSMBlobTokenGenerator(
   container: BlobContainerName,
   endpoint: EndpointURL,
   workspaceId: WorkspaceId,
+  containerResourceId: ContainerResourceId,
   wsmClient: WorkspaceManagerApiClientProvider) extends BlobTokenGenerator {
 
   def generateAccessToken: Try[AzureSasCredential] = Try {
-    val token = wsmClient.getControlledAzureResourceApi.createAzureStorageContainerSasToken(
+    val controlledAzureResourceApi = AzureCredentials(None).getAccessToken
```

Comment:
This is a bit of an inversion of the logic we had discussed. Here we try to get a fresh token from the environment and if that fails, try the default one provided as a dev argument. I chose this mostly for ease of implementation however I can revisit if we prefer that the dev arg overrides any other token

Reply:
I think we want the dev one to override if it exists because locally we *can* get a token, it's just not one that gives us permission to make this query to WSM. Maybe my understanding is wrong, though, is that different from what you've observed?